test-cases/tests/upgrade/n07-verify-rds-and-elasticache-db-instance-types-are-successfull.md

```diff
@@ -3,11 +3,13 @@ targets:
   - 2.5.0
 ---
 
-# N07 - Verify RDS and Elasticache DB instance types are successfully migrated
+# N07 - Verify RDS and Elasticache DB instance types are upgraded during maintenance
```

Comment:
the test case name should also be reflected in the file name. Run `npm run rename` (or `npm rename` - not sure of the top of my head). Also make sure that the linter doesn't complain, e.g. `npm run lint`